### PR TITLE
Update Puppetfile for 1.22.0-RC2

### PR DESCRIPTION
**Pull request by tbrisker**
Update puppet-foreman version for kubevirt support and locked
puppet-postgres to < 6 to avoid conflicts with katello-candlepin that
still requires < 6

### CODE REVIEW

**Review by ekohl — CHANGES_REQUESTED** [review]
I also did a pulp release, could you include that as well?

**Comment by tbrisker** [comment]
@ekohl updated.

File: Puppetfile.lock [hunk 1]
```diff
@@ -1,8 +1,8 @@
 FORGE
   remote: https://forgeapi.puppetlabs.com
   specs:
-    camptocamp-systemd (2.3.0)
-      puppetlabs-inifile (>= 1.6.0, < 3.0.0)
+    camptocamp-systemd (2.4.0)
```
Comment (ekohl) [inline Puppetfile.lock]: Normally I'd be conservative and leave this out

File: Puppetfile.lock [hunk 2]
```diff
@@ -46,15 +46,15 @@ FORGE
       puppetlabs-stdlib (>= 4.13.0, < 6.0.0)
     puppet-extlib (3.1.0)
       puppetlabs-stdlib (>= 4.6.0, < 6.0.0)
-    puppet-mongodb (2.4.1)
```
Comment (ekohl) [inline Puppetfile.lock]: Same here

File: Puppetfile.lock [hunk 3]
```diff
@@ -46,15 +46,15 @@ FORGE
...
-      puppetlabs-apt (>= 2.1.0, < 7.0.0)
-      puppetlabs-stdlib (>= 4.13.0, < 6.0.0)
-    puppet-selinux (1.6.1)
-      puppetlabs-stdlib (>= 4.13.1, < 6.0.0)
-    puppet-squid (2.0.0)
-      puppet-selinux (>= 1.0.0, < 2.0.0)
+    puppet-mongodb (3.0.0)
+      puppetlabs-apt (>= 2.1.0, < 8.0.0)
+      puppetlabs-stdlib (>= 4.25.0, < 7.0.0)
+    puppet-selinux (2.0.0)
```
Comment (ekohl) [inline Puppetfile.lock:52]: This one I would include because it contains an EL8 fix

File: Puppetfile.lock [hunk 4]
```diff
@@ -46,15 +46,15 @@ FORGE
...
+      puppetlabs-stdlib (>= 4.18.0, < 7.0.0)
+    puppet-squid (2.2.0)
```
Comment (ekohl) [inline Puppetfile.lock:54]: This one is needed to allow the newer selinux

File: Puppetfile.lock [hunk 5]
```diff
@@ -68,19 +68,19 @@ FORGE
       puppetlabs-translate (>= 1.0.0, < 2.0.0)
     puppetlabs-firewall (1.15.3)
       puppetlabs-stdlib (>= 4.0.0, < 6.0.0)
-    puppetlabs-inifile (2.5.0)
+    puppetlabs-inifile (3.0.0)
```
Comment (ekohl) [inline Puppetfile.lock]: I'd also leave this out

File: Puppetfile.lock [hunk 6]
```diff
@@ -68,19 +68,19 @@ FORGE
...
       puppetlabs-translate (>= 1.0.0, < 2.0.0)
-    puppetlabs-mysql (8.1.0)
-      puppetlabs-stdlib (>= 3.2.0, < 6.0.0)
+    puppetlabs-mysql (9.0.0)
```
Comment (ekohl) [inline Puppetfile.lock]: And this

File: Puppetfile.lock [hunk 7]
```diff
@@ -68,19 +68,19 @@ FORGE
...
+      puppetlabs-stdlib (>= 3.2.0, < 7.0.0)
       puppetlabs-translate (>= 1.0.0, < 2.0.0)
     puppetlabs-postgresql (5.12.1)
       puppetlabs-apt (>= 2.0.0, < 7.0.0)
       puppetlabs-concat (>= 1.1.0, < 6.0.0)
       puppetlabs-stdlib (>= 4.13.1, < 6.0.0)
-    puppetlabs-puppetdb (7.1.0)
+    puppetlabs-puppetdb (7.2.0)
```
Comment (ekohl) [inline Puppetfile.lock]: And this :)

**Review by ekohl — CHANGES_REQUESTED** [review]
A more complete list. Normally I whitelist changes that I do want, which is just a few this time.

**Comment by tbrisker** [comment]
@ekohl i hope i got it right this time :)

**Review by ekohl — APPROVED** [review]

**Pull request merged**